DMTN-276.tex

```diff
@@ -53,88 +56,263 @@
-  }\label{fig:affected_pixels}
+  The decay of the persistence with consecutive dark images. 
+  The blue points are the measured average counts of the spot (left) and trail (right).
+  The red line shows the model fit with the parameters shown in the upper right.
+  }\label{fig:Run6_decay}
 \end{figure*}
 
+Figure \ref{fig:Pers_Flash_Tau} shows the values for \textit{A} and \textit{$\tau$} for all detectors and all exposures that showed persistence. 
+This figure shows that while \textit{$\tau$} is roughly the same for all images (between 10--20 seconds), the amplitude of the effect changes.
+There also is a relationship between flashtime/the brightest of the spot and the persistence measured signal.
 
-Since this effect will not be easily removed, I looked into how persistence will affect LSST like images and measurements on the objects. 
-I adopted the same model as desribed above for the persistence and its trail and added it into subsquent images. 
-Figure \ref{fig:ex_persistence} shows an example of a DC2 image and the model persistence applied to it. 
-I then plotted a histogram of the fraction of persistence affected pixels as seen in Figure \ref{fig:affected_pixels}. 
-This shows the number of images with the corresponidng fraction of affected pixels. 
-For most images, $<0.2\%$ of pixels are affected in the subsquent images.
+\begin{figure*}[!ht]
+  \centering
+  \includegraphics[width=0.95\textwidth, angle=0]{Persistence_Flash_Tau.png}
+  \caption{
+  The relationship between flash time (a proxy for a bright source) and amplitude (left) and tau (right) for the spot. 
+  The amplitude appears to be dependent on the flash time, while tau is invariant and is consistently between 10--20 seconds. 
+  }\label{fig:Pers_Flash_Tau}
+\end{figure*}
 
+\section{Effect of Persistence on DC2 Images}
 
-\section{Effect of Persistence on DC2 Objects}
+Though studies are currently being done to minimize or elminate persistence, both through hardware and processing, one of the next steps is to see how much persistence will affect LSST given its current rought characterization.
+To study these effects, we used LSST DC2 data and the rough characterization that all persistence has a \textit{$\tau$} of 15 seconds and all saturated pixels have a amplitude of 5 ADU\@.
+These numbers are similar to what we see with the pinhole data, which we feel will better represent on-sky data.
+As most of the characterization has been with the spot and not the trail, we use 5 different schema to determine the impact of the trail: assume the trail carries with it 100\%, 75\%, 50\%, 25\%, and 0\% of the spot persistence level.
+We only used one of the e2V detectors at random (detector number 88/R21-S21) for this analysis and then assume that this detector is representative of the other e2V detectors.
+We also only utilize single visit images to mimic what the effect would be as images would be coming from the observatory.
```

Comment:
Trying to say single visit instead of coadds. Though do you think it would be worth it to say single 30 second exposures? From what I saw, that's what all DC2 is but it might be worth it in case readers think that they are snaps. Right now I changed it to '... single-visit images with 30 second exposures ...'